src/skin/legacyskinparser.cpp

```diff
@@ -1067,6 +1065,49 @@ QWidget* LegacySkinParser::parseTrackProperty(const QDomElement& node) {
     return p;
 }
 
+QWidget* LegacySkinParser::parseTrackWidgetGroup(const QDomElement& node) {
+    QString channelStr = lookupNodeGroup(node);
+    const char* pSafeChannelStr = safeChannelString(channelStr);
+    BaseTrackPlayer* pPlayer = m_pPlayerManager->getPlayer(channelStr);
+
+    if (!pPlayer) {
```

Comment:
This is not a good Idea, because it happens in case of skin errors. 
Crashing Mixxx in this case is not a good idea. I will give a skin warning instead.